blueprint/office-365.md

```diff
@@ -1708,8 +1708,21 @@ Phishing protection | Enabled | Blocks forms from being shared or distributed if
 ## Microsoft Planner
 Microsoft Planner is a task planning and assignment tool which can integrate with other Microsoft 365 services such as Microsoft Teams. Planner leverages iCalendar publishing to allow planner users to add tasks into their calendar. When tasks are published via iCalendar, they are available to all users with the iCalendar URL. The users can be both internal and external to the organisation.
 
-Microsoft Planner Configuration applicable to all agencies and implementation types.
+Microsoft Planner Design Decisions for all agencies and implementation types.
 
 Decision Point | Design Decision | Justification
 --- | --- | ---
 Publishing through iCalendar feed | Disabled | To prevent users from sharing sensitive information via iCalendar feeds.
+
+## Viva Learning
+Viva Learning is a Microsoft Teams app which aims to assist users to find and access training from multiple content providers. Viva Learning content can be enabled from LinkedIn Learning, Microsoft Learn, and Microsoft 365 Training. Administrators can also configure a SharePoint site to host addition content.
```

Comment:
Just need a finishing line to explain why it is disabled in the table and a little guidance for users to know if it's safe to enable should they choose?

Reply:
Have updated with additional content.